packages/rocketchat-livechat/server/methods/saveAppearanceTexts.js

```diff
@@ -0,0 +1,29 @@
+Meteor.methods({
+	'livechat:saveAppearanceTexts'(settings) {
+		if (!Meteor.userId() || !RocketChat.authz.hasPermission(Meteor.userId(), 'view-livechat-manager')) {
+			throw new Meteor.Error('error-not-allowed', 'Not allowed', { method: 'livechat:saveAppearanceTexts' });
+		}
+
+		const validSettings = [
+			'Livechat_title',
+			'Livechat_offline_form_unavailable',
+			'Livechat_offline_message',
+			'Livechat_offline_success_message',
+			'Livechat_offline_title'
+		];
+
+		const valid = settings.every((setting) => {
+			return validSettings.indexOf(setting._id) !== -1;
+		});
+
+		if (!valid) {
+			throw new Meteor.Error('invalid-setting');
+		}
+
+		settings.forEach((setting) => {
+			RocketChat.settings.updateById(setting._id, setting.value);
```

Comment:
@sampaiodiego How would I go about saving and fetching these new text settings?
As you recommended, the new object structure is 
```const settingTexts = [
	{
		identifier: 'Livechat_title',
		lang: instance.chosenLanguage,
		text: _.trim(instance.title.get())
	},
	[...]
];
```
Do I need to create a new model? Is there a similar part of the software I could base it on?